website/src/documentation/transforms/java/aggregation/distinct.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: section
+title: "Distinct"
+permalink: /documentation/transforms/java/aggregation/distinct/
+section_menu: section-menu/documentation.html
+---
+<!--
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+# Distinct
+<table align="left">
+    <a target="_blank" class="button"
+        href="https://beam.apache.org/releases/javadoc/current/index.html?org/apache/beam/sdk/transforms/Distinct.html">
+      <img src="https://beam.apache.org/images/logos/sdks/java.png" width="20px" height="20px"
+           alt="Javadoc" />
+     Javadoc
+    </a>
+</table>
+<br>
+Produces a collection containing distinct elements of the input collection.
+
+Does not preserve the order of the input collection.
```

Comment:
Why calling out this here? I think `PCollection` is unordered and most transforms do not have any guarantee on ordering.